packages/accessibility/src/AccessibilityManager.ts

```diff
@@ -1,7 +1,9 @@
+import { CanvasRenderer } from '@pixi/canvas-renderer';
```

Comment:
We need to find a better solution. Importing CanvasRenderer like this is going to mess up the dependency tree for a lot of packages that need to know about the renderer. My suggestion is that we just use `AbstractRenderer | Renderer`. That should cover CanvasRenderer without having to import it explicitly.

Reply:
Ok I've updated this to remove `CanvasRenderer` for `AbstractRenderer`